test/staking/StakeManager.test.js

```diff
@@ -715,7 +715,7 @@ contract('StakeManager: Heimdall fee', async function (accounts) {
       // 2/3 majority vote
       await stakeManager.checkSignatures(
         22,
-        utils.bufferToHex(utils.keccak256(vote)),
+        utils.bufferToHex(utils.keccak256(Buffer.concat([utils.toBuffer('0x01'), utils.toBuffer(data)]))),
```

Comment:
We should add more test cases with 0x02, non-majority vote to test multiple branches.